_posts/2018-07-10-react-native-wdio.md

```diff
@@ -0,0 +1,358 @@
+---
+layout: post
+title: "React Native Application UI testing using WebDriverIO and Appium"
+date: 2018-07-09 22:58:15 -0800
+cover: /assets/images/cover-wdio.png
```

Comment:
Looks like this path doesn't match the file path you are checking in, which is _/assets/images/react-native-wdio/cover-wdio.png_. Either update the path here or move the file, whichever you prefer should be fine.